lib/migration.c

```diff
@@ -100,35 +89,30 @@ init_migration(const vfu_migration_callbacks_t * callbacks,
 
 void
 MOCK_DEFINE(migr_state_transition)(struct migration *migr,
-                                   enum migr_iter_state state)
+                                   enum vfio_user_device_mig_state state)
 {
     assert(migr != NULL);
     /* FIXME validate that state transition */
```

Comment:
is this FIXME still true?

Reply:
No, we validate it elsewhere. I'll remove it.